Frends.LDAP.SearchObjects/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## [2.2.0] - 2025-03-17
+### Added
+- Added support for custom attributes in the output. You can now define which attributes you want to get from the LDAP search.
```

Comment:
Wasn't this already present? If you were adding some attrs in Input, then only those were along for the ride? So this new parameter is kind of more a UI improvement for explicitly allowing the user to better understand how things work?

Reply:
I'll modify the change log. What I was trying to tell with this is that users can now specify certain attribute value types while searching attributes and you can also select if you want to fetch only the specified ones or if you just want to specify certain attribute types.